ehcache-impl/src/main/java/org/ehcache/impl/internal/loaderwriter/writebehind/operations/DeleteAllOperation.java

```diff
@@ -45,4 +48,13 @@ public void performOperation(CacheLoaderWriter<K, V> cacheLoaderWriter) throws B
     cacheLoaderWriter.deleteAll(entries);
   }
 
+  @Override
+  public Set<K> getKeys() {
```

Comment:
I wonder if this should be typed as `Iterable<K>` so that we can implement it as a wrapper on the `entries` iterable?